packages/sdk/src/types/runtime/actor.ts

```diff
@@ -97,6 +97,8 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
     private _attachment: Attachment;
     private _lookAt: LookAt;
     private _materialId = ZeroGuid;
+    private _grabbable = false;
+    private _grab = new DiscreteAction();

```

Comment:
Will be moving the _grab out of this area and prefixing with $ as it should not be observed.

Reply:
No need for that since we don't observe the entire actor, just parts of it.

---
In reply to: [261903170](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/233#discussion_r261903170) [](ancestors = 261903170)